frontend/src/services/toast-message.ts

```diff
@@ -0,0 +1,57 @@
+import { useToast } from "primevue/usetoast";
+import { ToastSeverity, type ToastSeverityOptions } from 'primevue/api';
+import type { ToastMessageOptions } from "primevue/toast";
+
+const defaultLifeTime = 3000;
+
+export const messageService = () => {
+  const toast = useToast();
+
+  const messageInfo = (title: string, detail: string, lifeTime?: number) => {
+    message({
+      severity: 'info',
+      summary: title,
+      detail: detail,
+      life: lifeTime ?? defaultLifeTime
+    })
+  }
+
+  const messageSuccess = (title: string, detail: string, lifeTime?: number) => {
+    message({
+      severity: 'success',
+      summary: title,
+      detail: detail,
+      life: lifeTime ?? defaultLifeTime
+    })
+  }
+
+  const messageWarning = (title: string, detail: string, lifeTime?: number) => {
+    message({
+      severity: 'warn',
+      summary: title,
+      detail: detail,
+      life: lifeTime ?? defaultLifeTime,
+    })
+  }
+
+  const messageError = (title: string, detail: string, lifeTime?: number) => {
+    message({
+      severity: 'error',
+      summary: title,
+      detail: detail,
+      life: lifeTime ?? defaultLifeTime
+    })
+  }
+
+  const message = (message: ToastMessageOptions) => {
```

Comment:
Questo a cosa serve?

Reply:
lo avevo fatto in modo da esporre un metodo sulla stessa interfaccia ma completamente configurabile senza preset

Reply:
Ah perfetto